src/prisma/schema.prisma

```diff
@@ -38,3 +40,16 @@ model Profile {
   website                  Json?
   tdaGradYearBannerVisible Json
 }
+
+model Post {
+  id                 Int   @id @default(autoincrement())
+  userId             Int?   @unique
```

Comment:
I don't think we want the `@unique` here, because I believe that would mean that users could only have one post. The reason we have userId unique on the profile is so because users cannot have more than one profile.